web/src/js/components/Authentication/Authentication.js

```diff
@@ -228,6 +228,11 @@ class Authentication extends React.Component {
           // authentication.
           // https://github.com/gladly-team/tab/issues/589
           this.props.fetchUser()
+
+          const urlParams = getUrlParameters()
+          if (urlParams.reauth === 'true') {
+            await this.onAuthProcessCompleted()
```

Comment:
Would like thoughts if I should decompose `onAuthProcessCompleted` into just a `redirect` function.

Reply:
@jedtan Could you explain what you mean here? If you'd just like to move it from a method into its own function—sure!

Reply:
That question makes no sense when i reread it haha. I actually think my original question is moot - i was just thinking of splitting out a separate redirect function that could be called from both `onAuthProcessCompleted` and here (replacing this `onAuthProcessCompleted` call). But now that I think about it, better to reuse the same generic "do this after auth is done" logic here.